java/time-based-query/README.md

```diff
@@ -1,21 +1,36 @@
-# Time Based Query
-Apply a time-based parameter to a feature layer query
+# Time based query
 
-![Time Based Query App](time-based-query.png)
+Query data using a time extent. 
+
+![Image of time-based query](time-based-query.png)
+
+## Use case
+
+This workflow can be used to return records that are between a specified start and end date. For example, records of Canada goose sightings over time could be queried to only show sightings during the winter migration time period.
 
 ## How to use the sample
-Simply run the app.
+
+Run the sample, and a subset of records will be displayed on the map.
 
 ## How it works
-1. Create an instance of `ServiceFeatureTable` in manual cache mode.
-1. Create a feature layer from the table and add it to the map's operational layers.
-1. Then create an instance of `QueryParameters` and specify a time extent with a start and end time. 
-1. Finally, use `populateFromServiceAsync(query, clearCache, outFields)` method present on `ServiceFeatureTable` to populate features based on the specified time interval.
+
+1. An instance of `ServiceFeatureTable` is created by passing a URL to the REST endpoint of a time-enabled service. Time-enabled services will have TimeInfo defined in the service description. This information is specified in ArcMap or ArcGIS Pro prior to publishing the service.
+2. Create an instance of `ServiceFeatureTable` in manual cache mode, so you can control how and when the feature table is populated with data.
```

Comment:
I think this step would be better worded as "set the feature request mode of ..." as we have already created the service feature table.